tests/Datasets/Control.php

```diff
@@ -1,17 +1,22 @@
 <?php
 
+namespace r\Tests\Datasets;
+
+//use function r\db;
+
+
 class Control extends Dataset
 {
-    protected function create()
+    public function populate()
+    {
+    }
+    public function truncate()
     {
-        r\dbCreate('Control')->run($this->conn);
-        r\db('Control')->tableCreate('t1')->run($this->conn);
-    }    
-    
-    protected function delete()
+        \r\db($this->db)->table('t1')->delete()->run($this->conn);
+    }
+
+    public function create()
     {
-        r\dbDrop('Control')->run($this->conn);
+        \r\db($this->db)->tableCreate('t1')->run($this->conn);
     }
 }
-
-?>
```

Comment:
What's the reason you removed the closing `?>` tags in all files? Is that a common practice? It looks wrong to me.